app/views/hyrax/base/_representative_media.html.erb

```diff
@@ -0,0 +1,9 @@
+<% if presenter.iiif_viewer? %>
+  <% if defined?(viewer) && viewer %>
```

Comment:
I'm confused what `viewer` is in this case.  Might you mean `iiif_viewer_display`?

Reply:
I think it's [this](https://github.com/samvera/hyrax/blob/main/app/views/hyrax/base/show.html.erb#L18) `viewer`